Marlin/src/lcd/language/language_de.h

```diff
@@ -394,6 +409,7 @@
   #define MSG_FILAMENT_CHANGE_INIT_1          _UxGT("Bitte warten...")
   #define MSG_FILAMENT_CHANGE_UNLOAD_1        _UxGT("Auswerfen...")
   #define MSG_FILAMENT_CHANGE_INSERT_1        _UxGT("Laden und Klick")
+  #define MSG_FILAMENT_CHANGE_HEAT_1          _UxGT("Klicke zum Heizen")
   #define MSG_FILAMENT_CHANGE_HEATING_1       _UxGT("Heizen...")
   #define MSG_FILAMENT_CHANGE_LOAD_1          _UxGT("Laden...")
   #define MSG_FILAMENT_CHANGE_PURGE_1         _UxGT("Entleeren...")
```

Comment:
what about "Reinigen..." instead of "Entleeren..."?

Reply:
Entleeren ist okay, meine Meinung.
"Reinigen" is clean.